.gitignore

```diff
@@ -38,3 +38,6 @@ build
 docs_source/extract
 docs_source/resources.mk
 docs_source/public
+
+# examples
+terraform.tfstate*
```

Comment:
nope, it ignores them all. This behavior is the reason I renamed `converge` to `converge-hcl`